README.md

```diff
@@ -158,6 +158,31 @@ As you may have noticed there are several ways how to specify arguments. Here's
   * values specified on CLI
   * interactive mode arguments
 
+## Answer File Schema
+
+The answer file schema can be described using Puppet types as such:
+
+```
+Hash[
+String $puppet_class,
+  Enum[true, false, Hash[String, Variant[String, Boolean, Integer, Array, Hash]]] $parameters
+]
+```
```

Comment:
I'm not sure this makes sense in this format. Or here. What I wanted to do was document the current schema to allow then to document a Version 2 schema. I considered doing this in a comment in the file itself.